website/docs/docs/deploy/deploy-jobs.md

```diff
@@ -13,57 +13,63 @@ You can use deploy jobs to build production data assets. Deploy jobs make it eas
 - Job run details, including run timing, [model timing data](#model-timing), and [artifacts](/docs/deploy/artifacts)
 - Detailed run steps with logs and their run step statuses
 
-You can create a deploy job and configure it to run on [scheduled days and times](#schedule-days) or enter a [custom cron schedule](#custom-cron-schedules).
+You can create a deploy job and configure it to run on [scheduled days and times](#schedule-days) or enter a [custom cron schedule](#cron-schedule).
 
 
 ## Prerequisites
 
-- You must have a dbt Cloud account and [Developer seat license](/docs/cloud/manage-access/seats-and-users). If you don't, you can [sign up](https://www.getdbt.com/signup/) for a [free account](https://www.getdbt.com/pricing/). 
+- You must have a [dbt Cloud account](https://www.getdbt.com/signup/) and [Developer seat license](/docs/cloud/manage-access/seats-and-users).
+    - For the [Trigger on job completion](#trigger-on-job-completion) feature, your dbt Cloud account must be on the [Team or Enterprise plan](https://www.getdbt.com/pricing/).
```

Comment:
Keeping the format consistent
```suggestion
    - You must have a dbt Cloud account on the [Team or Enterprise plan](https://www.getdbt.com/pricing/) for the [Trigger on job completion](#trigger-on-job-completion) feature.
```

Reply:
keeping as is. following prior art here https://docs.getdbt.com/docs/deploy/ci-jobs#prerequisites 

and i think i prefer the feature name to be in the front in this case